services/opsgenie.rb

```diff
@@ -84,5 +83,14 @@ def log(msg)
         puts(msg)
       end
     end
+
+    def url
+      uri = URI.parse("htpts://api.opsgenie.com/v1/json/appoptics")
```

Comment:
`https`

Reply:
Hah! Good 👁

Reply:
And how the heck are these tests passing?!